test/pylib/scylla_cluster.py

```diff
@@ -353,7 +362,7 @@ async def install(self) -> None:
         self.logger.info("installing Scylla server in %s...", self.workdir)
 
         # Cleanup any remains of the previously running server in this path
-        shutil.rmtree(self.workdir, ignore_errors=True)
+        await async_rmtree(self.workdir, ignore_errors=True)
```

Comment:
@kbr-scylla @kostja Please review.

Reply:
should we submit this separately?
we may want to backport this particular patch to older releases as well to reduce flakiness there

Reply:
It's enough for it to be separate commit to backport

Reply:
@xtrey please review this commit (https://github.com/scylladb/scylladb/commit/5ca54a6e8818b72a0634d0123cc110529d994354)